source/_integrations/goodwe.markdown

```diff
@@ -0,0 +1,55 @@
+---
+title: GoodWe
+description: Instructions on how to connect your GoodWe Inverter to Home Assistant.
+ha_category:
+  - Sensor
+  - Energy
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_release: 2022.2
+ha_codeowners:
+  - "@mletenay"
+ha_domain: goodwe
+ha_platforms:
+  - sensor
+---
+
+The GoodWe integration will poll a [GoodWe](http://www.goodwe.com/) solar inverter over the local network and present its runtime values as sensors in Home Assistant.
+
+It works with GoodWe ET, EH, BT, BH, ES, EM, DT, MS, D-NS, XS and BP families of inverters. Different inverter families/models expose different sets of sensors, the newer models have usually broader support.
+
+<div class='note'>
+If you can't communicate with the inverter despite your model is listed above, it is possible you have an old ARM firmware version. You should ask manufacturer support to upgrade your ARM firmware (not just inverter firmware) to be able to communicate with the inverter via UDP.
+
+It may work on other inverter families as well, as long as they listen on UDP port 8899 and respond to one of supported communication protocols. In general, if you can communicate with the inverter via official mobile application (PvMaster, SolarGo), it is very likely the integration will work too.
```

Comment:
```suggestion
It may work on other inverter families as well, as long as they listen on UDP port 8899 and respond to one of the supported communication protocols. In general, if you can communicate with the inverter via an official mobile application (PvMaster, SolarGo), it is very likely the integration will work too.
```